src/activities/categorization/Categorization.qml

```diff
@@ -123,73 +122,73 @@ ActivityBase {
                     spacing: 5
                     width: dialogActivityConfig.width
                     height: dialogActivityConfig.height
-                    property alias easyModeBox: easyModeBox
-                    property alias mediumModeBox: mediumModeBox
-                    property alias expertModeBox: expertModeBox
+                    property alias instructionsBox: instructionsBox
+                    property alias scoreBox: scoreBox
+                    property alias categoryImageBox: categoryImageBox
 
                     GCDialogCheckBox {
-                        id: easyModeBox
+                        id: instructionsBox
                         width: column.width - 50
-                        text: qsTr("Instructions and score visible")
-                        checked: (items.mode == "easy") ? true : false
-                        exclusiveGroup: configOptions
+                        text: qsTr("Instructions visible")
+                        checked: items.instructionsVisible
                         onCheckedChanged: {
-                            if(easyModeBox.checked) {
-                                items.mode = "easy"
-                                menuScreen.iAmReady.visible = false
-                            }
+                            items.instructionsVisible = instructionsBox.checked
+                            menuScreen.iAmReady.visible = ((!items.scoreVisible && !items.instructionsVisible) || (!items.instructionsVisible && items.scoreVisible))
```

Comment:
it should be done automatically with the binding, no need to reset it (and break the binding)